### PR TITLE
remove unused code

### PR DESCRIPTION
**Pull request by CyberDem0n**
And fix some codacy issues

### CODE REVIEW

File: tests/test_api.py [hunk 1]
```diff
@@ -103,7 +103,7 @@ def test_do_GET(self):
             MockRestApiServer(RestApiHandler, b'GET /master')
         MockRestApiServer(RestApiHandler, b'GET /master')
 
-    def test_do_OPTIONS(self):
+    def test_do_OPTIONS(*args):
```
Comment (CyberDem0n) [inline tests/test_api.py]: Codacy Issue found: [Method has no argument](https://www.codacy.com/app/cyberdemn/patroni/file/2060706155/issues/source?bid=3127594&fileBranchId=3135461#l106)

File: tests/test_api.py [hunk 2]
```diff
@@ -87,7 +87,7 @@ def __init__(self, Handler, path):
 @patch('ssl.wrap_socket', Mock(return_value=0))
 class TestRestApiHandler(unittest.TestCase):
 
-    def test_do_GET(self):
+    def test_do_GET(*args):
```
Comment (CyberDem0n) [inline tests/test_api.py]: Codacy Issue found: [Method has no argument](https://www.codacy.com/app/cyberdemn/patroni/file/2060706155/issues/source?bid=3127594&fileBranchId=3135461#l90)

File: tests/test_api.py [hunk 3]
```diff
@@ -117,14 +117,14 @@ def test_do_OPTIONS(self):
...
+    def test_do_GET_patroni(*args):
         MockRestApiServer(RestApiHandler, b'GET /patroni')
 
-    def test_basicauth(self):
+    def test_basicauth(*args):
         MockRestApiServer(RestApiHandler, b'POST /restart HTTP/1.0')
         MockRestApiServer(RestApiHandler, b'POST /restart HTTP/1.0\nAuthorization:')
 
-    def test_do_POST_restart(self):
+    def test_do_POST_restart(*args):
```
Comment (CyberDem0n) [inline tests/test_api.py]: Codacy Issue found: [Method has no argument](https://www.codacy.com/app/cyberdemn/patroni/file/2060706155/issues/source?bid=3127594&fileBranchId=3135461#l127)

File: tests/test_api.py [hunk 4]
```diff
@@ -140,10 +140,10 @@ def test_do_POST_reinitialize(self, dcs):
         with patch.object(MockHa, 'schedule_reinitialize', Mock(return_value=None)):
             MockRestApiServer(RestApiHandler, request)
         cluster.leader.name = 'test'
-        MockRestApiServer(RestApiHandler, request)
+        self.assertIsNotNone(MockRestApiServer(RestApiHandler, request))
 
     @patch('time.sleep', Mock())
-    def test_RestApiServer_query(self):
+    def test_RestApiServer_query(*args):
```
Comment (CyberDem0n) [inline tests/test_api.py]: Codacy Issue found: [Method has no argument](https://www.codacy.com/app/cyberdemn/patroni/file/2060706155/issues/source?bid=3127594&fileBranchId=3135461#l146)

File: tests/test_api.py [hunk 5]
```diff
@@ -117,14 +117,14 @@ def test_do_OPTIONS(self):
             makefile.return_value.flush = Mock(side_effect=socket.error("foo"))
             MockRestApiServer(RestApiHandler, b'OPTIONS / HTTP/1.0')
 
-    def test_do_GET_patroni(self):
+    def test_do_GET_patroni(*args):
```
Comment (CyberDem0n) [inline tests/test_api.py]: Codacy Issue found: [Method has no argument](https://www.codacy.com/app/cyberdemn/patroni/file/2060706155/issues/source?bid=3127594&fileBranchId=3135461#l120)

File: tests/test_api.py [hunk 6]
```diff
@@ -117,14 +117,14 @@ def test_do_OPTIONS(self):
...
         MockRestApiServer(RestApiHandler, b'GET /patroni')
 
-    def test_basicauth(self):
+    def test_basicauth(*args):
```
Comment (CyberDem0n) [inline tests/test_api.py]: Codacy Issue found: [Method has no argument](https://www.codacy.com/app/cyberdemn/patroni/file/2060706155/issues/source?bid=3127594&fileBranchId=3135461#l123)

File: tests/test_ctl.py [hunk 7]
```diff
@@ -69,20 +69,16 @@ def setUp(self):
...
-        assert c is None
+        # MockCursor returns pg_is_in_recovery as false
+        self.assertIsNone(get_cursor(get_cluster_initialized_with_leader(), role='replica'))
 
-        c = get_cursor(get_cluster_initialized_with_leader(), role='any')
-        assert c is not None
+        self.assertIsNotNone(get_cursor(get_cluster_initialized_with_leader(), role='any'))
 
-    def test_output_members(self):
+    def test_output_members(*args):
```
Comment (CyberDem0n) [inline tests/test_ctl.py]: Codacy Issue found: [Method has no argument](https://www.codacy.com/app/cyberdemn/patroni/file/2060706158/issues/source?bid=3127594&fileBranchId=3135461#l81)

File: tests/test_ctl.py [hunk 8]
```diff
@@ -1,25 +1,19 @@
...
 from patroni.ctl import ctl, members, store_config, load_config, output_members, post_patroni, get_dcs, \
     wait_for_leader, get_all_members, get_any_member, get_cursor, query_member, configure
-from patroni.ha import Ha
 from patroni.etcd import Etcd, Client
-from test_ha import get_cluster_initialized_without_leader, get_cluster_initialized_with_leader, \
-    get_cluster_initialized_with_only_leader, MockPostgresql, MockPatroni, run_async, \
-    get_cluster_not_initialized_without_leader
+from patroni.exceptions import PatroniCtlException
+from psycopg2 import OperationalError
+from requests.exceptions import ConnectionError
```
Comment (CyberDem0n) [inline tests/test_ctl.py]: Codacy Issue found: [Redefining built-in 'ConnectionError'](https://www.codacy.com/app/cyberdemn/patroni/file/2097116845/issues/source?bid=3127594&fileBranchId=3135461#l13)

**Comment by alexeyklyukin** [comment]
+1

**Pull request merged**